### PR TITLE
test ResNet decorated by dygraph_to_static_output in static mode.

### PR DESCRIPTION
**Pull request by liym27**
Test model ResNet transformed to static graph.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_resnet.py [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+            end_time = time.time()
+            if batch_id % 2 == 0:
+                print( "epoch %d | batch step %d, loss %0.3f, acc1 %0.3f, acc5 %0.3f, time %f" % \
+                       ( epoch, batch_id, total_loss / total_sample, \
+                         total_acc1 / total_sample, total_acc5 / total_sample, end_time-start_time))
+            if batch_id == 10:
+                break
+
+
+def train_resnet_in_dygraph_mode():
```
Comment (Aurelius84) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_resnet.py]: This function will not be executed that will reduce the code coverage. Please remove the function and add it in next PR.

Comment (liym27) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/test_resnet.py]: Thanks. Done.

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Pull request merged**